src/image/components/runtime.tsx

```diff
@@ -46,7 +47,7 @@ export const Runtime: React.FC<IProps> = ({ authoredState }) => {
 
   const handleClick = () => {
     const allowUpscale = scaling === "fitWidth";
-    const modalImageUrl = highResUrl || url;
+    const modalImageUrl = highResUrl || defaultImageUrl;
```

Comment:
This change seems unnecessary -- it's essentially the same as `highResUrl || url || highResUrl`.